### ISSUE TITLE
Cradle fails when no "request" is configured in the options

### ISSUE THREAD

**Issue by mickaeltr** [flatiron/cradle#302]
Hello,

Since cradle 0.7.0, my application fails with:
```
TypeError: Cannot set property 'method' of undefined
    at node_modules/cradle/lib/cradle.js:317:30
    at Array.forEach (native)
    at node_modules/cradle/lib/cradle.js:315:24
    at Array.forEach (native)
    at Object.cradle.merge (node_modules/cradle/lib/cradle.js:314:10)
    at Connection.cradle.Connection.rawRequest (node_modules/cradle/lib/cradle.js:162:22)
```

I fixed it with some extra configuration: `cradle.setup({request: {}});`

EDIT: this workaround does not seem to work very well though

**Comment by panuhorsmalahti** [comment]
Damn. This must be caused by https://github.com/flatiron/cradle/commit/51c9142c678c681d06f4b7ea0023653a7ba0ed92

@Freyert, do you have time for a quick fix? I'll see if I can fix it today.

**Comment by panuhorsmalahti** [comment]
Released 0.7.1, @mickaeltr could you try if it fixes your problem?

**Comment by mickaeltr** [comment]
It seems to work fine now, thanks.
